tlsfuzzer/expect.py

```diff
@@ -179,6 +179,8 @@ def process(self, state, msg):
                     self.extensions[ext_id](state, ext)
                 if ext_id == ExtensionType.extended_master_secret:
                     state.extended_master_secret = True
+                if ext_id == ExtensionType.encrypt_then_mac:
+                    state.encrypt_then_mac = True
```

Comment:
Do you mean resetting it same as `extended_master_secret` at `expect.py` line 168? Then at `ChangeCipherSpecGenerator()` change encrypt-then-mac state for record layer?